README.md

```diff
@@ -479,7 +522,7 @@ $ xmake l
   }
 ```
 
-More builtin plugins exist, please see: [Builtin plugins](https://xmake.io/#/plugin/builtin_plugins)
+To see a list of bultin plugs, please visit [Builtin plugins](https://xmake.io/#/plugin/builtin_plugins).
```

Comment:
Not sure about using  `plugs` here. maybe `plugins`?

Reply:
Sorry, typo on my part.